Source/AwsCommonRuntimeKit/Utilities.swift

```diff
@@ -25,6 +26,21 @@ extension String {
     var awsByteCursor: aws_byte_cursor {
         return aws_byte_cursor_from_c_str(self.asCStr())
     }
+
+    func md5(allocator: Allocator = defaultAllocator, truncate: Int = 0) -> String? {
```

Comment:
Maybe it would be good to change the name of this function to:
```
base64EncodedMD5
```

Reply:
also needs the `public`modifier

Reply:
updated